sdk/servicebus/service-bus/test/internal/serviceBusClient.spec.ts

```diff
@@ -1,359 +1,854 @@
 // Licensed under the MIT license.
 
-import { extractReceiverArguments, ServiceBusClient } from "../../src/serviceBusClient";
+import { EnvironmentCredential } from "@azure/identity";
 import chai from "chai";
-import { ServiceBusSessionReceiverOptions } from "../../src/models";
-import { entityPathMisMatchError } from "../../src/util/errors";
+import chaiAsPromised from "chai-as-promised";
+import * as dotenv from "dotenv";
+import Long from "long";
 import {
-  createConnectionContextForConnectionString,
-  createConnectionContextForTokenCredential
-} from "../../src/constructorHelpers";
-import { TokenCredential } from "@azure/core-http";
-import { ConnectionContext } from "../../src/connectionContext";
-import { createConnectionContextForTestsWithSessionId } from "./unittestUtils";
+  isServiceBusError,
+  ProcessErrorArgs,
+  ServiceBusClient,
+  ServiceBusError,
+  ServiceBusSessionReceiver
+} from "../../src";
+import { ServiceBusSender } from "../../src";
+import { DispositionType, ServiceBusReceivedMessage } from "../../src/serviceBusMessage";
+import { getReceiverClosedErrorMsg, getSenderClosedErrorMsg } from "../../src/util/errors";
+import { EnvVarNames, getEnvVars, isNode } from "../public/utils/envVarUtils";
+import { checkWithTimeout, TestClientType, TestMessage } from "../public/utils/testUtils";
 import {
-  ServiceBusSessionReceiver,
-  ServiceBusSessionReceiverImpl
-} from "../../src/receivers/sessionReceiver";
-import { AbortController } from "@azure/abort-controller";
-const assert = chai.assert;
-
-const allLockModes: ("peekLock" | "receiveAndDelete")[] = ["peekLock", "receiveAndDelete"];
-
-describe("serviceBusClient unit tests", () => {
-  // the options type is used for our tests but it's not _hugely_ important, only
-  // that it can and does get returned verbatim from whatever options slot
-  // we pass.
-  // So if we add other options types there's no need to generate a whole
-  // new set of tests for it. :)
-  const sessionReceiverOptions: ServiceBusSessionReceiverOptions = {};
-
-  const testEntities = [
-    { queue: "thequeue", entityPath: "thequeue" },
-    {
-      topic: "thetopic",
-      subscription: "thesubscription",
-      entityPath: "thetopic/Subscriptions/thesubscription"
-    }
-  ];
+  createServiceBusClientForTests,
+  EntityName,
+  ServiceBusClientForTests,
+  testPeekMsgsLength,
+  getRandomTestClientTypeWithSessions,
+  getRandomTestClientTypeWithNoSessions
+} from "../public/utils/testutils2";
+import { ServiceBusReceiver, ServiceBusReceiverImpl } from "../../src/receivers/receiver";
+
+const should = chai.should();
+chai.use(chaiAsPromised);
+
+dotenv.config();
+
+const noSessionTestClientType = getRandomTestClientTypeWithNoSessions();
+const withSessionTestClientType = getRandomTestClientTypeWithSessions();
+
+describe("ServiceBusClient live tests", () => {
+  let sbClient: ServiceBusClient;
+
+  describe("Create ServiceBusClient", function(): void {
```

Comment:
I see many additions, but I believe you have only moved the tests around.

Reply:
Yes but these show up in git as additions. Sorry @richardpark-msft